.storybook/preview-head.html

```diff
@@ -0,0 +1,7 @@
+<style>
+  html,
+  body,
+  #storybook-root {
```

Comment:
This style was lost in the upgrade from SB 6 -> 7 and is needed to show the map in a few stories